src/skills-builder/skills-builder-context/SkillsBuilderProvider.jsx

```diff
@@ -1,12 +1,24 @@
 import React, { createContext, useReducer, useMemo } from 'react';
 import PropTypes from 'prop-types';
 import reducer, { skillsInitialState } from '../data/reducer';
+import { useAlgoliaSearch } from '../utils/search';
 
 export const SkillsBuilderContext = createContext();
 
 export const SkillsBuilderProvider = ({ children }) => {
   const [state, dispatch] = useReducer(reducer, skillsInitialState);
-  const value = useMemo(() => ([state, dispatch]), [state]);
+
+  const [searchClient, productSearchIndex, jobSearchIndex] = useAlgoliaSearch();
+
+  const value = useMemo(() => ({
+    state,
+    dispatch,
+    algolia: {
+      searchClient,
+      productSearchIndex,
+      jobSearchIndex,
+    },
+  }), [state, searchClient, productSearchIndex, jobSearchIndex]);
```

Comment:
We will need to reach for `searchClient` at least one more time in the app. Storing it in context gives us easy access to it; rather than passing it down through various parent components.